.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [master]
```

Comment:
should we also add the `release` branch?

Reply:
I wasn't aware of the `release` branch. I can assume what it is for but probably we can skip having a separate branch altogether as we'll go with tags/releases concept going forward. Let's talk more about this in tomorrow's call?